### PR TITLE
Userclass server side

### PR DESCRIPTION
**Pull request by tjsparks5**
The user's classes should now be properly saving to the database and loading from the database (no more manually entering in stuff into the database).
To test this please:
(1) Save a list of classes on the myclasses page (add courses then hit submit).
(2) Refresh and see if they load again on the myclasses page (it used to just reload blank).
(3) Then go to the chat and see if the classes load into the chat view in under "my classes".

The user's classes are stored in the database in the "userclasses" collection if you would like to look at the database.

**Note**: You may notice that clicking submit multiple times with the same list or a different list of courses you have added will save multiple entries into the db. I have not implemented update yet but I will be working on that, this will allow only one entry per user, but for now multiple will happen.

### CODE REVIEW

**Review by Bakushin10 — APPROVED** [review]
the structure of this code is very hard to follow. userModule and classModule are mixed up. Usually it should be fixed before making the pull request but @tjsparks5  is also aware of this, and (hopefully) will be working on it in different branch so I will still accept. 

-- bug -- 
* you can create multiple different data sets for classes which you should not be able to. And, it only loads the first one. so we need to implement some kind of function to prevent this from happening.

**Comment by yotoo2920** [comment]
I found exactly those bugs:
1- only the first time are the classes being load after pressing submit
2- duplicates 
Note: We agreed to work on this after the sprint two presentation

**Review by yotoo2920 — APPROVED** [review]
We talk about the bugs. We agreed to go for it for now

**Comment by tjsparks5** [comment]
Shin closed to create new test branch. Reopening.
